var/spack/repos/builtin/packages/fpocket/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Fpocket(Package):
```

Comment:
Switch to `MakefilePackage` base class, rename `patch` to `edit`, and drop `install` method.

Reply:
done